samza-sql/src/main/java/org/apache/samza/sql/translator/QueryTranslator.java

```diff
@@ -190,9 +245,9 @@ public RelNode visit(LogicalAggregate aggregate) {
     sendToOutputStream(outputSystemStream, streamAppDescriptor, translatorContext, node, queryId);
   }
 
-  private void sendToOutputStream(String sinkStream, StreamApplicationDescriptor appDesc, TranslatorContext context, RelNode node, int queryId) {
+  private void sendToOutputStream(String sinkStream, StreamApplicationDescriptor appDesc, TranslatorContext translatorContext, RelNode node, int queryId) {
     SqlIOConfig sinkConfig = sqlConfig.getOutputSystemStreamConfigsBySource().get(sinkStream);
```

Comment:
We don't have an operator for SendTo, I think we should represent sendTo as an operator and can we have the following metrics

1. processing time
2. event rate.

Reply:
done